x/auth/signing/sign_mode_handler.go

```diff
@@ -20,7 +20,19 @@ type SignModeHandler interface {
 
 	// GetSignBytes returns the sign bytes for the provided SignMode, SignerData and Tx,
 	// or an error
-	GetSignBytes(ctx context.Context, mode signing.SignMode, data SignerData, tx sdk.Tx) ([]byte, error)
+	GetSignBytes(mode signing.SignMode, data SignerData, tx sdk.Tx) ([]byte, error)
+}
+
+// SignModeHandlerWithContext is like SignModeHandler, with a new GetSignBytes
```

Comment:
Until when do we want to say backward compatible?
Can we add a TODO comment that says to delete this when < 0.48 is deprecated?

Reply:
Added.